AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -1299,7 +1299,7 @@ - (void)testFlushQueueIteratesThroughArrayRecursively {
   [tokenValidityArray addObject:token1];
   [tokenValidityArray addObject:token2];
   [tokenValidityArray addObject:token3];
-  NSArray<id<MSLog>> *logsForToken1 = [self getValidMockLogArrayForDate:date1 andCount:0];
+  NSArray<id<MSLog>> *logsForToken1 = [self getValidMockLogArrayForDate:date1 andCount:3];
```

Comment:
I am confused in this test about how it verifies that the correct logs are sent with the correct token.